.github/workflows/build-and-publish.yml

```diff
@@ -59,3 +59,23 @@ jobs:
         github_actor: ${{ secrets.LNB_ACTOR }}
         github_token: ${{ secrets.LNB_TOKEN }}
 
+  test-build:
+    name: "Test published image"
+    needs: build
+    runs-on: ubuntu-22.04
+    if: github.repository == 'hpcc-systems/HPCC-Platform'
+    env:
+      WF_YAML_FILE: az-bvt.yml
+    steps:
+      - name: "Trigger external azure test code"
+        run: |
+          community_tag=$( echo ${{ github.ref }} | cut -d'/' -f3 )
+          hpcc_version=$( echo $community_tag | sed 's/community_//' | sed 's/-[0-9]$//' )
```

Comment:
These variables are used in the curl call.

Reply:
Looks good